benchmarks/encoder/utils.py

```diff
@@ -49,7 +57,7 @@ def gen_traces(config):
             span_name = random.choice(span_names)
             resource = random.choice(resources)
             service = random.choice(services)
-            with Span(span_name, resource=resource, service=service, parent_id=parent_id) as span:
+            with _Span(span_name, resource=resource, service=service, parent_id=parent_id) as span:
```

Comment:
`gen_span` ?

Reply:
I think such a function could be useful across benchmark scenarios but I didn't want to do that here.

Reply:
oh 🤦🏻 right